### ISSUE TITLE
Deep Manifold Traversal: Changing Labels with Convolutional Features

### ISSUE THREAD

**Issue by zdx3578** [alexjc/neural-doodle#66]
http://arxiv.org/abs/1511.06421    http://arxiv.org/pdf/1511.06421v3

Many tasks in computer vision can be cast as a "label changing" problem, where the goal is to make a semantic change to the appearance of an image or some subject in an image in order to alter the class membership. Although successful task-specific methods have been developed for some label changing applications, to date no general purpose method exists. Motivated by this we propose deep manifold traversal, a method that addresses the problem in its most general form: it first approximates the manifold of natural images then morphs a test image along a traversal path away from a source class and towards a target class while staying near the manifold throughout. The resulting algorithm is surprisingly effective and versatile. It is completely data driven, requiring only an example set of images from the desired source and target domains. We demonstrate deep manifold traversal on highly diverse label changing tasks: changing an individual's appearance (age and hair color), changing the season of an outdoor image, and transforming a city skyline towards nighttime.

**Comment by alexjc** [comment]
I saw this a few months ago, great research! I think this will be a separate open source project when implemented, though.

**Comment by zdx3578** [comment]
1  in that paper  new r3 verson 'Deep Manifold Traversal' author say will open source in github,but now has not yet open .
2 DGN can Arithmetic on faces https://github.com/Newmu/dcgan_code;

**Comment by zdx3578** [comment]
http://arxiv.org/abs/1603.08155    Perceptual Losses for Real-Time Style Transfer and Super-Resolution

Justin Johnson, Alexandre Alahi, Li Fei-Fei
(Submitted on 27 Mar 2016)
We consider image transformation problems, where an input image is transformed into an output image. Recent methods for such problems typically train feed-forward convolutional neural networks using a \emph{per-pixel} loss between the output and ground-truth images. Parallel work has shown that high-quality images can be generated by defining and optimizing \emph{perceptual} loss functions based on high-level features extracted from pretrained networks. We combine the benefits of both approaches, and propose the use of perceptual loss functions for training feed-forward networks for image transformation tasks. We show results on image style transfer, where a feed-forward network is trained to solve the optimization problem proposed by Gatys et al in real-time. Compared to the optimization-based method, our network gives similar qualitative results but is three orders of magnitude faster. We also experiment with single-image super-resolution, where replacing a per-pixel loss with a perceptual loss gives visually pleasing results.
